src/app/core/services/config-options/config-options.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@angular/core';
+
+import { ConfigOptions } from '../../models/config-options';
+
+@Injectable({
+  providedIn: 'root',
+})
+
+export class ConfigOptionsService {
+  configOptions: ConfigOptions;
+
+  constructor() {
+    this.configOptions = {};
+  }
+
+  setOptions({ id, login, email }: ConfigOptions) {
```

Comment:
method type
won't write anymore this comment, recheck whole app yourself

Reply:
fixed